docs/training/index.rst

```diff
@@ -0,0 +1,64 @@
+#######################
+ Model Developer Guide
+#######################
+
+.. raw:: html
+
+   <div class="landing">
+      <div class="tiles-flex">
+         <div class="tile-container">
+            <a class="tile" href="../architecture/index.html">
```

Comment:
Is it intentional to have this tile point to architecture/index.html, and also have a top-level link to architecture/index.html?

I find that disorienting, since I'm used to docs having a hierarchical structure to them.  When I go from top-level page to Model Developer Guide, having another doc that points to what was a peer of Model Developer Guide in the top-level page feels weird

Reply:
this is intentional
- the welcome page is a starting point for new visitors
- the model developer guide is a landing page for a section
- sometimes the tiles will have two homes (and maybe also slightly different titles depending on the audience and purpose)
- i have more plans for landing pages so this is an interim step to get me there